src/content/developers/tutorials/send_token_etherjs.md

```diff
@@ -0,0 +1,71 @@
+## send Token Using ethers.js(5.0)
+
+this send_token() is not only for ether transfer 
+but also for other erc20 tokens too.
```

Comment:
Would try to introduce this more thoroughly. Double check grammar as well, as this should be an educational walkthrough. 
Wrap inline code, such as `send_token()` in single backticks to style it appropriately and help readability.